packages/gatsby-source-wordpress/src/normalize.js

```diff
@@ -234,6 +235,40 @@ exports.mapTagsCategoriesToTaxonomies = entities =>
     return e
   })
 
+exports.searchReplaceContentUrls = async function ({ entities, searchReplace }) {
+
+  if (!Array.isArray(searchReplace) && searchReplace.length !== 2) {
+    return entities
+  }
+
+  const [search, replace] = searchReplace
+
+  const _blacklist = [
+    `_links`,
+    `__type`,
+  ]
+
+  const blacklistProperties = function (obj = {}, blacklist = {}) {
+    for (var i = 0; i < blacklist.length; i++) {
+        eval(`delete obj.${blacklist[i]}`)
```

Comment:
I used eval here because I wanted to account for a scenario where the blacklist goes deeper than one level for example: `'content.rendered'`.

If you think we will never have to blacklist data in the object deeper than level one I can make this change :)

Reply:
If we extend the blacklist we could change the code.